deployment/helm/templates/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  #namespace: {{ .Values.settings.env }}
+  labels:
+    app.kubernetes.io/name: {{ .Values.settings.name }}
+    app.kubernetes.io/part-of: {{ .Values.settings.name }}
+    helm.sh/chart: {{ include "chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  allowed-hosts: {{ .Values.security.allowedHosts | quote }}
+  module: {{ .Values.settings.module }}
```

Comment:
I'd rename this to `django_settings_module` to stay in line with common practices (and mimic the environment variable that applies this)